editor/assets/stylesheets/_variables.scss

```diff
@@ -19,6 +19,7 @@ $light-gray-300: #edeff0;
 $light-gray-200: #f3f4f5;
 $light-gray-100: #f8f9f9;
 $white: #fff;
+$red: #dc3232;
```

Comment:
Probably. I'm not sure where the current hex for $alert-red comes from, but #dc3232 is the new standard red in core (Which is also used on the notices).